prediction_market_agent/db/pinecone_handler.py

```diff
@@ -109,15 +112,22 @@ def deduplicate_markets(markets: list[OmenMarket]) -> list[OmenMarket]:
 
         return list(unique_market_titles.values())
 
-    def insert_all_omen_markets_if_not_exists(
-        self, created_after: DatetimeUTC | None = None
+    def update_markets(self) -> None:
+        """We use the agent's run to add embeddings of new markets that don't exist yet in the
+        vector DB."""
+        self.insert_open_omen_markets_if_not_exists()
+
+    def insert_open_omen_markets_if_not_exists(
+        self, start_timestamp: int | None = None
     ) -> None:
         subgraph_handler = OmenSubgraphHandler()
         markets = subgraph_handler.get_omen_binary_markets_simple(
             limit=sys.maxsize,
             filter_by=FilterBy.NONE,
+            created_after=DatetimeUTC.fromtimestamp(start_timestamp)
```

Comment:
```suggestion
            created_after=DatetimeUTC.to_datetime_utc(start_timestamp)
```

`to_datetime_utc` is our method that is doing some more fancy stuff, check it out there 🙏